src/google/protobuf/compiler/command_line_interface.cc

```diff
@@ -168,7 +168,12 @@ bool VerifyDirectoryExists(const string& path) {
 // directories listed in |filename|.
 bool TryCreateParentDirectory(const string& prefix, const string& filename) {
   // Recursively create parent directories to the output file.
+#if defined(_WIN32)
+  // on Windows, both '/' and '\' are valid path separators
+  std::vector<string> parts = Split(filename, "/\\", true);
```

Comment:
Can you use this code path always? I.e., splitting on "/\\" even on linux. I don't think we want to support "\" as part of the filename at all.